### PR TITLE
[IN-2442] use ansible builtins to mirror own git repo

### PR DESCRIPTION
**Pull request by bitrise-tom**
- [IN-2442] homebrew install javas with cask option
- [IN-2442] use ansible builtins to mirror own git repo

** This should eventually move into the Brew role **
this is a stage 1 exit to use ansible's built in modules
to configure the local mirror of brew cask-versions

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

[IN-2442]: https://bitrise.atlassian.net/browse/IN-2442
[IN-2442]: https://bitrise.atlassian.net/browse/IN-2442

### CODE REVIEW

File: roles/java/tasks/main.yml [hunk 1]
```diff
@@ -39,32 +41,26 @@
...
     name: adoptopenjdk8
     state: present
+    install_options: cask
 
 - name: brew install openjdk11
-  homebrew_cask:
+  homebrew:
     name: adoptopenjdk11
     state: present
+    install_options: cask
```
Comment (peterheja-bitrise) [inline roles/java/tasks/main.yml:63]: b47697a42f07845645ce00b08cddd4e96c742848

File: roles/java/tasks/main.yml [hunk 2]
```diff
@@ -39,32 +41,26 @@
...
   homebrew_tap:
     name: AdoptOpenJDK/openjdk
     state: present
 
 - name: brew install openjdk8
-  homebrew_cask:
+  homebrew:
     name: adoptopenjdk8
     state: present
+    install_options: cask
```
Comment (peterheja-bitrise) [inline roles/java/tasks/main.yml:57]: b47697a42f07845645ce00b08cddd4e96c742848

**Review by peterheja-bitrise — CHANGES_REQUESTED** [review]
Drop commit b47697a42f07845645ce00b08cddd4e96c742848

File: roles/java/tasks/main.yml (hunk 1 above)
Comment (bitrise-tom) [reply roles/java/tasks/main.yml:63]: when i merge the first PR, these will vanish :)

File: roles/java/tasks/main.yml (hunk 2 above)
Comment (bitrise-tom) [reply roles/java/tasks/main.yml:57]: when i merge the first PR, these will vanish :)

**Review by fehersanyi-bitrise — DISMISSED** [review]

**Review by fehersanyi-bitrise — APPROVED** [review]

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Pull request merged**